docs/prompting/zero_shot/emotion_prompting.md

```diff
@@ -1,7 +1,46 @@
 ---
-title: ""
+title: "Emotion Prompting"
 description: ""
-keywords: ""
+keywords: "prompt, prompting, prompt engineering, llm, gpt, openai, emotion prompting, model, AI, python, instructor, zero-shot"
 ---
 
-[wip]
+Emotion prompting<sup><a href="https://arxiv.org/abs/2307.11760">1</a></sup> uses phrases that have psychological relevance to humans in the LLM prompt. This may lead to improved performance on benchmarks and open-ended text generation.<sup><a href="https://arxiv.org/abs/2406.06608">\*</a></sup>
+
+```python
```

Comment:
I do feel like this is a bit too complex. Ideally we'd like a simple example that can show how emotional prompting can increase the quality/verbosity of the outputs 

```
from openai import OpenAI
import instructor
from pydantic import BaseModel, Field


class Recomendation(BaseModel):
    recomendation: str = Field(
        description="A recomendation for a user based on the content of the prompt"
    )


client = instructor.from_openai(OpenAI())

for include_emotion in [True, False]:
    emotional_prompt = (
        "Take pride in your work and give it your best. Your commitment to excellence sets you apart"
        if include_emotion
        else ""
    )
    summary = client.chat.completions.create(
        model="gpt-4o",
        response_model=Recomendation,
        messages=[
            {
                "content": f"Tell me about the best way to get better at cooking.{emotional_prompt}",
                "role": "user",
            }
        ],
    )
    print(summary.recomendation)
```

What do you think about something like this? Also it's usful to add the output as 

```
print(output) # This output is here
```